tests/assets/disable_polling.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: fleet.cattle.io/v1alpha1
+kind: GitRepo
+metadata:
+  name: test-disable-polling
+  annotations:
+    {}
+    #  key: string
+  labels:
+    {}
+    #  key: string
```

Comment:
This is not required. Also, on line no. 14-15 and line no. 9 as well. <-- Suggestion can be implemented in next PR.

Reply:
Changes applied.
Removed also commented `-string`line.
I left however 
` correctDrift:
    enabled: false` given that by default it was created in Yaml when playing it and I am afraid it may take the true value. It can be tweaked later